test-apps/display-test-app/src/frontend/App.ts

```diff
@@ -54,6 +54,8 @@ import { TerrainDrapeTool } from "./TerrainDrapeTool";
 import { SaveImageTool } from "./SaveImageTool";
 import { BingTerrainMeshProvider } from "./BingTerrainProvider";
 import { OpenRealityModelSettingsTool } from "./RealityModelDisplaySettingsWidget";
+import { ElectronRendererAuthorization } from "@itwin/electron-authorization/lib/cjs/ElectronRenderer";
```

Comment:
Curious why the explicit cjs import?

Reply:
We dont have a single barrel file for the electron pkgs (core-electron and electron-auth) due to splitting things as needed for the main vs renderer process. I believe previously when we did have them in a single barrel file, we were accidentally pulling in electron main code into the frontend when we shouldnt have